### ISSUE TITLE
logo px值修改不生效

### ISSUE THREAD

**Issue by gh-Devin** [mrhelloboy/seven#1]
我在hugo.html里面修改后，不生效，页面还是默认56px，想问下怎么修改？

 [Params.logo]
    # max height not greater than 56 (px). default: 56 (px)
    customLogoHeight = '265'
    customLogoWidth = '400'

**Comment by gh-Devin** [comment]
另外如果想首页增加banner轮播图的话，你之前有做过么？或者未来有计划做吗

**Comment by mrhelloboy** [comment]
> 我在hugo.html里面修改后，不生效，页面还是默认56px，想问下怎么修改？
> 
> [Params.logo] # max height not greater than 56 (px). default: 56 (px) customLogoHeight = '265' customLogoWidth = '400'

你好，是因为设置了最大高度不能超过56px的原因。如果你不希望有这个，可以删掉 `layouts/partials/header.html` 里面的代码（在14行位置）：
```
{{ if ge $logoH 56 }}
    {{ $logoH = 56 }}
{{ end }}
```

**Comment by mrhelloboy** [comment]
> 另外如果想首页增加banner轮播图的话，你之前有做过么？或者未来有计划做吗

@gh-Devin banner轮播图不难，但轮播图可能会破坏页面的简洁布局，我暂时还没计划添加该功能，或许可以相互探讨一下😁

**Comment by gh-Devin** [comment]
> > 我在hugo.html里面修改后，不生效，页面还是默认56px，想问下怎么修改？
> > [Params.logo] # max height not greater than 56 (px). default: 56 (px) customLogoHeight = '265' customLogoWidth = '400'
> 
> 你好，是因为设置了最大高度不能超过56px的原因。如果你不希望有这个，可以删掉 `layouts/partials/header.html` 里面的代码（在14行位置）：
> 
> ```
> {{ if ge $logoH 56 }}
>     {{ $logoH = 56 }}
> {{ end }}
> ```

我layouts/partials/就只有一个extra_social.html

**Comment by gh-Devin** [comment]
git clone https://github.com/mrhelloboy/hugo-theme-seven-demo.git

**Comment by gh-Devin** [comment]
> > 另外如果想首页增加banner轮播图的话，你之前有做过么？或者未来有计划做吗
> 
> @gh-Devin banner轮播图不难，但轮播图可能会破坏页面的简洁布局，我暂时还没计划添加该功能，或许可以相互探讨一下😁

应该不太会，其实就是在现有首页上面加个轮播图，其实也还好，哈哈。让他更像一个web而不是blog，哈哈哈，比如首页可以有个banner图和部分news链接过来，有些导航下面其实只是需要一个介绍页面类似于about me，有些导航下面可以是blog/news风格

**Comment by mrhelloboy** [comment]
> > > 我在hugo.html里面修改后，不生效，页面还是默认56px，想问下怎么修改？
> > > [Params.logo] # max height not greater than 56 (px). default: 56 (px) customLogoHeight = '265' customLogoWidth = '400'
> > 
> > 
> > 你好，是因为设置了最大高度不能超过56px的原因。如果你不希望有这个，可以删掉 `layouts/partials/header.html` 里面的代码（在14行位置）：
> > ```
> > {{ if ge $logoH 56 }}
> >     {{ $logoH = 56 }}
> > {{ end }}
> > ```
> 
> 我layouts/partials/就只有一个extra_social.html

不是 clone demo 仓库，是 clone 本仓库然后修改源码，然后在自己站点上的 `hugo.toml` 修改 import 的主题：
```toml
[module]
[[module.imports]]
  path = 'github.com/your_name/seven'
```

**Comment by mrhelloboy** [comment]
> > > 另外如果想首页增加banner轮播图的话，你之前有做过么？或者未来有计划做吗
> > 
> > 
> > @gh-Devin banner轮播图不难，但轮播图可能会破坏页面的简洁布局，我暂时还没计划添加该功能，或许可以相互探讨一下😁
> 
> 应该不太会，其实就是在现有首页上面加个轮播图，其实也还好，哈哈。让他更像一个web而不是blog，哈哈哈，比如首页可以有个banner图和部分news链接过来，有些导航下面其实只是需要一个介绍页面类似于about me，有些导航下面可以是blog/news风格

主题的初衷其实是 blog，如果要弄到成你想要的风格，得 fork 代码然后自己魔改了

**Comment by gh-Devin** [comment]
> > > > 另外如果想首页增加banner轮播图的话，你之前有做过么？或者未来有计划做吗
> > > 
> > > 
> > > @gh-Devin banner轮播图不难，但轮播图可能会破坏页面的简洁布局，我暂时还没计划添加该功能，或许可以相互探讨一下😁
> > 
> > 
> > 应该不太会，其实就是在现有首页上面加个轮播图，其实也还好，哈哈。让他更像一个web而不是blog，哈哈哈，比如首页可以有个banner图和部分news链接过来，有些导航下面其实只是需要一个介绍页面类似于about me，有些导航下面可以是blog/news风格
> 
> 主题的初衷其实是 blog，如果要弄到成你想要的风格，得 fork 代码然后自己魔改了

哈哈哈，对，我准备魔改下，哈哈，banner图咋改，还希望你给指点指点

**Comment by gh-Devin** [comment]
> > > > 我在hugo.html里面修改后，不生效，页面还是默认56px，想问下怎么修改？
> > > > [Params.logo] # max height not greater than 56 (px). default: 56 (px) customLogoHeight = '265' customLogoWidth = '400'
> > > 
> > > 
> > > 你好，是因为设置了最大高度不能超过56px的原因。如果你不希望有这个，可以删掉 `layouts/partials/header.html` 里面的代码（在14行位置）：
> > > ```
> > > {{ if ge $logoH 56 }}
> > >     {{ $logoH = 56 }}
> > > {{ end }}
> > > ```
> > 
> > 
> > 我layouts/partials/就只有一个extra_social.html
> 
> 不是 clone demo 仓库，是 fork 本仓库然后修改源码，然后在自己站点上的 `hugo.toml` 修改 import 的主题：
> 
> ```toml
> [module]
> [[module.imports]]
>   path = 'github.com/your_name/seven'
> ```

除了改这里，还需要改别的地方吧？比如go.mod,go.sum啥的，我改了go.mod,gosum,首页就不显示了，哈哈哈
